SPECS/leveldb/leveldb.spec

```diff
@@ -0,0 +1,208 @@
+%undefine __cmake_in_source_build
+
+Summary:        A fast and lightweight key/value database library by Google
+Name:           leveldb
+Version:        1.22
+Release:        3%{?dist}
+License:        BSD
+URL:            https://github.com/google/leveldb
+Vendor:         Microsoft
+Distribution:   Mariner
+Source0:        %{url}/archive/%{version}/%{name}-%{version}.tar.gz
+
+# available in https://github.com/fusesource/leveldbjni/blob/leveldb.patch
+Patch0001:      0001-Allow-leveldbjni-build.patch
+# https://github.com/fusesource/leveldbjni/issues/34
+# https://code.google.com/p/leveldb/issues/detail?id=184
+# Add DB::SuspendCompactions() and DB:: ResumeCompactions() methods
+Patch0002:      0002-Added-a-DB-SuspendCompations-and-DB-ResumeCompaction.patch
+# Cherry-picked from Basho's fork
+Patch0003:      0003-allow-Get-calls-to-avoid-copies-into-std-string.patch
+# https://groups.google.com/d/topic/leveldb/SbVPvl4j4vU/discussion
+Patch0004:      0004-bloom_test-failure-on-big-endian-archs.patch
+
+BuildRequires:  cmake
+#Provided by gcc package
+#BuildRequires:  gcc-c++
+BuildRequires:  gcc
+BuildRequires:  make
+BuildRequires:  snappy-devel
+BuildRequires:  sqlite-devel
+
+%description
+LevelDB is a fast key-value storage library written at Google that provides an
+ordered mapping from string keys to string values.
+
+%package devel
+Summary:        Development files for %{name}
+# Provided by cmake
+#Requires:      cmake-filesystem
+Requires:       cmake
+Requires:       %{name}%{?_isa} = %{version}-%{release}
+
+%description devel
+%{summary}.
+
+%prep
+%autosetup -p1
+
+cat > %{name}.pc << EOF
+prefix=%{_prefix}
+exec_prefix=${prefix}
+libdir=%{_libdir}
+includedir=%{_includedir}
+
+Name: %{name}
+Description: %{summary}
+Version: %{version}
+Libs: -l%{name}
+EOF
+
+%build
+%cmake
+%make_build
+
+%install
+%make_install
+
+mkdir -p %{buildroot}%{_libdir}/pkgconfig
+cp -a %{name}.pc %{buildroot}%{_libdir}/pkgconfig/
+
+%check
+%ctest
```

Comment:
Do we actually support the `ctest` macro?

Reply:
Removed the macro. Replaced it with 'ctest -V %{?_smp_mflags}'